docs/fullDocDraft.md

```diff
@@ -0,0 +1,361 @@
+# G4CMP Documentation
+
+## Overview
+
+G4CMP is a framework designed to add to the Geant4 toolkit for use in condensed
+matter and low-energy physics. Developed for the low-temperature community,
+G4CMP is capable of modeling several physics processes relevant to phonon and
+charge collection at cryogenic temperatures
+[.](../README.md) These include anisotropic phonon
```

Comment:
What is the "[.]" here?  Does this make just the period at the end of the sentence a link to the README?  I don't think that's what we want, here or elsewhere.

Reply:
Yes, that's what it does. This is just the direct porting over of the Google Doc comments for references. Would it be better if I wrote something like `[[1]](../README.md)` after the sentence instead?